.gitignore

```diff
@@ -0,0 +1 @@
+tmp
```

Comment:
remember add new line at end of the each file

Reply:
Sure